skbeam/core/utils.py

```diff
@@ -1115,7 +1115,8 @@ def twotheta_to_q(two_theta, wavelength):
     Returns
     -------
     q : array
-        An array of :math:`q` values
+        An array of :math:`q` values in the inverse of the units
+        of ``wavelength``
```

Comment:
IIRC `q` here is in inverse angstroms so we can be more specific about the units of `wavelength`

Reply:
That is not automatically true. Some experiments are done in inverse nanometers

Reply:
Oh yeah my bad. That's definitely correct 🦆 . Ignore my comment above too about the inverse of this function

Reply:
if you want to be pedantic, it's technically radians per length as opposed to cycles per wavelength (this has always confused me)

Reply:
here, quick [wiki search](https://en.wikipedia.org/wiki/Wavenumber) for instance:
> Wavenumber can be used to specify quantities other than spatial frequency. In optical spectroscopy, it is often used as a unit of temporal frequency assuming a certain speed of light. In this context, it is the number of cycles—not radians—per unit length.